guide/12-enrich-data-with-thematic-information/part6_country_enrich.ipynb

```diff
@@ -0,0 +1,1115 @@
+{
```

Comment:
<p>The enrich capability in Business Analyst requires polygon areas to be used for <a href="https://developers.arcgis.com/rest/GeoEnrichment/api-reference/data-apportionment.htm" target="_blank">apportioning demographic data</a> to the input geographies. In this example, we are using addresses for potential store locations. While geocoding can be used to get the location of the stores, the <code>enrich</code> method still requires polygonal areas to be able to apportion demographic data. </p><p>First, we can use geocoding to get the geographic location of all the stores. Since we are using a Pandas DataFrame, we can take advantage of data manipulation and schema pruning. We will concatenate the components of the address into a concise column for geocoding. Then, we will reduce the geocoding response to just the columns we need for our subsequent analysis steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>